### PR TITLE
Bees will escape from bags when opened by someone

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

Opening a bag or container that has scooped up hostile simple mobs (bees) will cause them to pop out at the person who opened it.

## Why It's Good For The Game

You can have a bag full of bees, but you can't prank people with them, and that's sad

image

This allows for devious bee related pranks and that's good

image

## Changelog
:cl: Melbert
add: Bees, if placed in a container, will pop out at the next person to open it
/:cl:

### CODE REVIEW

File: code/modules/mob/living/inhand_holder.dm [hunk 1]
```diff
@@ -90,10 +101,10 @@
 
 /obj/item/clothing/head/mob_holder/destructible/Destroy()
 	if(held_mob)
-		release(FALSE, TRUE)
+		release(FALSE, TRUE, TRUE)
 	return ..()
 
-/obj/item/clothing/head/mob_holder/release(del_on_release = TRUE, delete_mob = FALSE)
+/obj/item/clothing/head/mob_holder/destructible/release(del_on_release = TRUE, display_messages = TRUE, delete_mob = FALSE)
```
Comment (MrMelbert) [inline code/modules/mob/living/inhand_holder.dm:107]: I'm pretty sure this was a bad copy-paste

**Comment by MimicFaux** [comment]
Beautiful

**Comment by brotherbeyondd** [comment]
> Beautiful

Beeutiful

**Comment by Iamgoofball** [comment]
This is a huge nerf to beekeeping.

**Comment by Doosteroni** [comment]
WHERE AM I SUPPOSED TO KEEP MY BEES NOW!

**Comment by Iamgoofball** [comment]
^ this

**Comment by bluetshirtguy** [comment]
Where do you keep the bees? the station is your beehive.

**Comment by cacogen** [comment]
no not the bees

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/inhand_holder.dm [hunk 2]
```diff
@@ -72,6 +74,20 @@
...
+	if(held_mob)
+		for(var/i in held_mob.faction)
+			if(i == "hostile")
+				to_chat(finder, "<span class='warning'>\A [held_mob.name] pops out! </span>")
+				finder.visible_message("<span class='warning'>\A [held_mob.name] pops out of the container [finder] is opening!</span>", ignored_mobs = finder)
+				release(TRUE, FALSE)
+				return
+
+/obj/item/clothing/head/mob_holder/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0, datum/callback/callback, force, gentle = FALSE, quickstart = TRUE)
+	if(HAS_TRAIT(src, TRAIT_NODROP))
```
[the remark this answers is not in the document]
Comment (MrMelbert) [reply code/modules/mob/living/inhand_holder.dm:85]: actually, i'll just remove it and open and issue report

**Pull request merged**